### PR TITLE
[CHAD-8901] Updated all thermostatHeatingSetpoint to correct range

### PR DESCRIPTION
**Pull request by Bhavik-T**

### CODE REVIEW

File: drivers/SmartThings/zigbee-thermostat/profiles/base-thermostat-fanless.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatHeatingSetpoint
     version: 1
+    configuration:
+      - values:
+        key: "temperature.value"
```
Comment (bflorian) [inline drivers/SmartThings/zigbee-thermostat/profiles/base-thermostat-fanless.yml:11]: This needs to be `heatingSetpoint.value`

**Review by bflorian — CHANGES_REQUESTED** [review]
All of the `thermostatHeatingSetpoint` customizations need to be changed to `key: heatingSetpoint.value` because the attribute name is `heatingSetpoint`, not `temperature`. I did not comment on each file

**Review by greens — APPROVED** [review]

**Comment by ctowns** [comment]
I went ahead and just cherry-picked the commits from [CHAD-9353](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/268) and [CHAD-9316](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/269) to this PR since they are all dealing with just setting the embedded device profile for the thermostatHeatingSetpoint. I'll close those once this goes in and then update the JIRA tickets to point at this PR instead of the old ones.

**Review by greens — APPROVED** [review]

**Comment by iplantemn** [comment]
@greens validated that the capability configs are in correct format

File: drivers/SmartThings/zigbee-thermostat/profiles/base-thermostat-fanless.yml [hunk 2]
```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatHeatingSetpoint
     version: 1
+    configuration:
```
Comment (lelandblue) [inline drivers/SmartThings/zigbee-thermostat/profiles/base-thermostat-fanless.yml:9]: configuration:   needs to be changed to   config: 


@ctowns Can you help with this please

**Comment by lelandblue** [comment]
Hey @ctowns We just heard from @iplantemn  that "configuration:" needs to change to "config:" Can you help us with that please?

File: drivers/SmartThings/zigbee-thermostat/profiles/base-thermostat-fanless.yml (hunk 2 above)
Comment (ctowns) [reply drivers/SmartThings/zigbee-thermostat/profiles/base-thermostat-fanless.yml]: Done!

**Review by greens — APPROVED** [review]

**Pull request merged**